medusa/tv/series.py

```diff
@@ -220,7 +220,8 @@ def __init__(self, indexer, indexerid, lang='', quality=None,
             {'episodes', 'next_aired', 'release_groups', 'imdb_info'})
 
         self.show_id = None
-        self._name = ''
+        self.name = ''
+        self._title = ''
```

Comment:
What's the point of having `self._title` if it's never set?

Reply:
good catch. I used it before. But decided to go with a different set up.
It's removed